src/components/groups/CreateGroup.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { Col, Row } from 'mdbreact';
+import { withI18n } from 'react-i18next';
+import { observer } from 'mobx-react';
+import { EditableItem } from '../controls/EditableItem/EditableItem';
+
+@observer
+class BaseCreateGroup extends React.Component {
+    state = {
+        value: ''
+    }
+
+    handleTextChange = (event) => {
+        const { name, value } = event.target;
+        this.setState({[name]: value});
+    }
+
+    render() {
+        return (
+            <div>
+                <Row>
+                    <Col md="12">
+                        <h2>הרשמה למחנה נושא 2018</h2>

```

Comment:
Looks good! but start using translation (based on `groupType` param)

Reply:
Done. Entered dummy english copy for now.